pages/index.js

```diff
@@ -1,32 +1,124 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
```

Comment:
remove these `eslint-disables`

Reply:
this will be done in the final code clean-up and also these eslint restrictions are needed else the travis CI will keep failing

Reply:
No, make the appropriate changes to your code so that the travis Ci doesn't fail